server.js

```diff
@@ -2,13 +2,25 @@
 
 const express = require('express');
 const puppeteer = require('puppeteer');
+var bodyParser = require('body-parser');
 
 // Constants
 const PORT = 80;
 const HOST = '0.0.0.0';
 
 // App
 const app = express();
+
+app.use(bodyParser.json({
+	limit: '50mb'
```

Comment:
Where does this value come from? I really feel that the limit should be insanely high, like 1-2GB, not that we would hit 1GB, just that we should not rule out the client trying to render a 200MB report with a 100 pages or something.

Reply:
We will increase if there is need. The biggest html I have created was 20MB and had all the SPIM data rendered in an Html table. At this level even the browser had a hard time rendering the output.